### PR TITLE
Fix error if category deleted #55

### PR DESCRIPTION
**Pull request by sanzeeb3**
Check if the `$food_term` variable if set and not empty, if not skip and continue the rest loop.

### CODE REVIEW

File: templates/layouts/grid-image.php [hunk 1]
```diff
@@ -30,6 +30,10 @@
 
 			$count = 1;
 			$food_term = get_term_by( 'id', $food_id, 'food_menu_cat' );
+			if( !isset( $food_term ) || empty( $food_term ) ) {
```
Comment (shivapoudel) [inline templates/layouts/grid-image.php]: I am confused why isset and empty check. Can't any one of there will do the same task. Also coding standard.

**Review by shivapoudel — CHANGES_REQUESTED** [review]
Some confusion left as review.

File: templates/layouts/grid-image.php (hunk 1 above)
Comment (shivapoudel) [reply templates/layouts/grid-image.php]: See https://codex.wordpress.org/Function_Reference/get_term_by and know what the returned value is and follow the procedural check.

Comment (sanzeeb3) [reply templates/layouts/grid-image.php]: okay. I thought `$food_term` can be 0 which is empty but set. `empty()` alone should do the task.

**Review by shivapoudel — APPROVED** [review]

**Comment by shivapoudel** [comment]
Cool thanks!

**Pull request merged**